go.mod

```diff
@@ -78,18 +79,18 @@ require (
 	github.com/prometheus/common v0.9.1
 	github.com/prometheus/tsdb v0.8.0 // indirect
 	github.com/rancher/dynamiclistener v0.2.1-0.20200418023342-52ede5ec9234
-	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200512184531-c395feca5b3b
+	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200327180925-6d88ca2af46f
 	github.com/rancher/machine v0.15.0-rancher25
-	github.com/rancher/norman v0.0.0-20200417060332-5c2a13443bb8
+	github.com/rancher/norman v0.0.0-20200326201949-eb806263e8ad
```

Comment:
Need to update this after merging https://github.com/rancher/norman/pull/364 with the commit from the release/v2.4 branch